apps/condo/domains/property/components/panels/Builder/MapConstructor.ts

```
@@ -676,115 +581,67 @@
     }
 
-    public setSelectedParkingUnit (unit: BuildingUnit): void {
-        if (this.isParkingUnitSelected(unit.id)) {
-            this.editMode = 'addParking'
-            this.selectedParkingUnit = null
+    public setSelectedUnit (unit: BuildingUnit): void {
+        if (this.isUnitSelected(unit.id)) {
+            this.selectedUnits = this.selectedUnits.filter(selectedUnit => selectedUnit.id !== unit.id)
+
+            if (this.selectedUnits.length === 0) {
+                return this.editMode = null
```

Biome (1.9.4)

[error] The function should not return a value because its return type is void.

The function is here:

'void' signals the absence of value. The returned value is likely to be ignored by the caller.

(lint/correctness/noVoidTypeReturn)